derive/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "sqlparser_derive"
+description = "proc macro for sqlparser"
+version = "0.1.0"
+authors = ["Andy Grove <andygrove73@gmail.com>"]
```

Comment:
TBH I'm not sure as well. Theoretically I'd say that the owner/creator should be the author, but since this is an expansion, not sure whether put both of you or just the @tustvold

Reply:
I think it is ok as is

Reply:
I changed it to "sqlparser authors" in https://github.com/sqlparser-rs/sqlparser-rs/pull/775